src/main/java/eu/fasten/maven/CheckMojo.java

```diff
@@ -95,6 +81,12 @@
     @Parameter
     private List<RiskAnalyzerConfiguration> configurations;
 
+    @Parameter(defaultValue = "https://api.fasten-project.eu/api")
```

Comment:
I would also add `property = "fastenApiUrl"` so that you can also set it in the command line through `-DfastenApiUrl=http://12.7.0.0.1:9090` which might actually be a more common use case than setting the property in the pom.

Reply:
Will add.